src/components/pages/lessons/overlay/anon-user-overlay.tsx

```diff
@@ -0,0 +1,318 @@
+    },
+    slug,
+    image,
+    path: `/courses/${slug}`,
+    description: '',
+    byline: '',
+  }
+}
+
+const FEATURE = 'anon-user-overlay'
+
+const transformHits = (hits: any, amount: integer) => {
+  if (!hits) return []
+  const slicedHits = hits.slice(0, amount)
+  return slicedHits.map((hit: any) => shapeHit(hit as any))
+}
+
+const ContinueCourseCard: React.FC<any> = ({
+  resource,
+  location,
+  className = 'border-none',
+  onClickRewatch,
+  router,
+  lesson,
+  nextLesson,
+  ...props
+}) => {
+  return (
+    <Card
+      {...props}
+      className={twMerge(
+        'bg-gray-800 bg-opacity-60 text-gray-200 shadow-sm rounded-lg flex flex-col items-center justify-center sm:text-left text-center',
+        className,
+      )}
+    >
+      <CardContent className="flex flex-col justify-center items-center">
+        <div className="hidden md:flex md:flex-col justify-center items-center shrink">
+          {resource.image && (
+            <Link href={resource.path}>
+              <a
+                onClick={() => {
+                  track('clicked resource', {
+                    resource: resource.path,
+                    linkType: 'image',
+                    location,
+                  })
+                }}
+                className="block flex-shrink-0 sm:w-auto m:w-24 w-36"
+                tabIndex={-1}
+              >
+                <CardPreview>
+                  <Image
+                    src={get(resource.image, 'src', resource.image)}
+                    width={180}
+                    height={180}
+                    layout="fixed"
+                    className="transition ease-in-out hover:scale-110 duration-100 object-cover rounded-md"
+                    alt={`illustration for ${resource.title}`}
+                  />
+                </CardPreview>
+              </a>
+            </Link>
+          )}
+          <CardHeader className="mx-16">
+            <Link href={resource.path}>
+              <a
+                onClick={() => {
+                  track('clicked resource', {
+                    resource: resource.path,
+                    linkType: 'text',
+                    location,
+                    feature: FEATURE,
+                  })
+                }}
+                className="inline-block hover:text-blue-600 dark:hover:text-blue-300 w-fit"
+              >
+                <h3 className="text-lg 3xl:text-xl font-bold leading-tighter dark:text-white dark:hover:text-blue-300 text-center">
+                  {resource.title}
+                </h3>
+              </a>
+            </Link>
+          </CardHeader>
+          <CardMeta className="text-xs justify-center text-gray-600 dark:text-gray-300 pb-2 pt-1 text-center">
+            {resource.byline}
+          </CardMeta>
+        </div>
+        <div className="flex flex-col items-center gap-y-2 3xl:mt-4">
+          {lesson?.collection?.path && (
+            <Link href={lesson.collection.path}>
```

Comment:
I honestly wouldn't expect for "Watch Full Course" to take me to course landing page.